runtime/src/lib.rs

```diff
@@ -225,7 +225,7 @@ parameter_types! {
         })
         .avg_block_initialization(AVERAGE_ON_INITIALIZE_RATIO)
         .build_or_panic();
-    pub const SS58Prefix: u16 = 2340;
+    pub const SS58Prefix: u16 = 1603;
```

Comment:
This value requires some explanation in comments or doc-line.

Reply:
oh, thanks! forgot to change it back.